README.md

```diff
@@ -29,6 +29,17 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
 - [Introducing Ably Asset Tracking - public beta now available](https://ably.com/blog/ably-asset-tracking-beta)
 - [Accurate Delivery Tracking with Navigation SDK + Ably Realtime Network](https://www.mapbox.com/blog/accurate-delivery-tracking)
 
+## Installation
+
+To use ably asset tracking in your app, install it as a dependency:
```

Comment:
This should be "Ably Asset Tracking" (capitalised first letters)

Reply:
Good point, fixed in 2c522e9